packages/connect-wallet-modal/src/helpers/useLocalStorage.ts

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+export function useLocalStorage(key: string, initialValue: any) {
+  // State to store our value
+  // Pass initial state function to useState so logic is only executed once
+  const [storedValue, setStoredValue] = useState(() => {
+    if (typeof window === 'undefined') {
+      return initialValue;
+    }
+    try {
+      // Get from local storage by key
+      const item = window.localStorage.getItem(key);
+      // Parse stored json or if none return initialValue
+      return item ? JSON.parse(item) : initialValue;
+    } catch (error) {
+      // If error also return initialValue
+      console.log(error);
```

Comment:
Why do we need to log it? Do we expect users to read developer console and report errors on their own?

Reply:
I guess it is better than do nothing and I don't know how to handle the error in other way. Most likely, the error will never happen